sunkit_image/radial.py

```diff
@@ -634,13 +630,14 @@ def _percentile_ranks_numpy_inplace(arr):
 @u.quantity_input(application_radius=u.R_sun, vignette=u.R_sun)
 def rhef(
     smap,
+    *,
     radial_bin_edges=None,
     application_radius=0 * u.R_sun,
     upsilon=0.35,
     method="numpy",
-    *,
-    vignette=1.5 * u.R_sun,
+    vignette=None,  # 1.5 * u.R_sun,
```

Comment:
These changes also need to be reverted or did you want to set this to be None?

Reply:
This turned out to be one of the bugs I found: when working with data that is much wider FOV than AIA, the vignette should be none or set to the user's input. So I think defaulting to None is best, unless we wanted some instrument-specific/ metadata-specific value to be set.

Reply:
Ah in that case yeah, set this to None. Only is a user sets the value, should we use it.